packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/PieChart/CustomLegendPlacement.2.doc.mdx

```diff
@@ -10,19 +9,17 @@ This is a simple Pie chart with custom legend placement.
 ## Getting Started
 
 - Install with [npmjs](https://www.npmjs.com):
-  - `npm install terra-graphs`
+  - `npm i @cerner/carbon-graphs --save-dev`
 
 ## Usage
-```jsx
-import React from 'react';
-import PieChart from '@cerner/terra-graphs/lib/components/Pie/PieChart';
+```js
+import Carbon from "@cerner/carbon-graphs/dist/js/carbon-graphs.js";
+import "@cerner/carbon-graphs/dist/css/carbon-graphs.css";
```

Comment:
Same for the other files.

```suggestion
import Carbon from '@cerner/carbon-graphs/dist/js/carbon-graphs.js';
import '@cerner/carbon-graphs/dist/css/carbon-graphs.css';
```

Reply:
Fixed in [this](https://github.com/cerner/terra-graphs/pull/188/commits/5823946c34ad6470c713cf80805334aeb0842be6#) commit.